app/presenters/notification_presenter.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 class NotificationPresenter < BasePresenter
-  def as_api_json(include_target=true)
+  def as_api_json
     data = base_hash
-    data = data.merge(target: target_json) if include_target && linked_object
+    data = data.merge(target: target_json) if linked_object && linked_object.is_a?(Post)
```

Comment:
Style/SafeNavigation: Use safe navigation (`&.`) instead of checking if an object exists before calling the method.